client/gutenberg/extensions/markdown/edit.jsx

```diff
@@ -19,10 +19,14 @@ const PANEL_EDITOR = 'editor';
 const PANEL_PREVIEW = 'preview';
 
 class MarkdownEdit extends Component {
+	input = null;
```

Comment:
Might wanna use the 'new' [`createRef` API](https://reactjs.org/docs/refs-and-the-dom.html#adding-a-ref-to-a-dom-element) instead? Seems more semantic here.

Reply:
Nice, hadn't seen that 👍

Reply:
That breaks right now, apparently because of this:

https://github.com/buildo/react-autosize-textarea/issues/102